ngraph/core/reference/include/ngraph/runtime/reference/group_convolution.hpp

```diff
@@ -34,6 +35,58 @@ namespace ngraph
     {
         namespace reference
         {
+            void validate_group_convolution_parameters(const Shape& in_shape,
```

Comment:
Added validation functions to check input ranks, group dimension, spatial dimensions of strides, dilations and pads. Additionally output shape is inferred and compared to output shape provided by the tensor descriptor. These checks are introduced because when using dynamic shapes some node validation check from `validate_and_infer_types` are by-passed.